### PR TITLE
Added "Swapping two variables" idiom

### PR DESCRIPTION
**Pull request by elizarov**

### CODE REVIEW

File: pages/docs/reference/idioms.md [hunk 1]
```diff
@@ -358,3 +358,13 @@ if (b == true) {
 }
 ```
 </div>
+
+### Swapping two variables
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+```kotlin
```
Comment (AlexanderPrendota) [inline pages/docs/reference/idioms.md]: add \n after div

Comment (elizarov) [reply pages/docs/reference/idioms.md]: After which div? I've added one more to the end of file. I don't see line breaks being 
used anywhere else in this file.

**Comment by ilya-g** [comment]
This looks like a clever hack rather than idiom 🤓 

I'd rather prefer to have something like `swap(::a, :b)` inline function in stdlib.

**Comment by elizarov** [comment]
I'd prefere it, too, but it would not really inline, as far as I can tell, and it will not work with array elements.

**Comment by elizarov** [comment]
On a side note, all idioms look like clever hacks until you start using them and learn to recognize them :)

**Pull request merged**